#!/usr/bin/python3
# raises name exception with custon msg
def raise_exception_msg(message=""):
    raise NameError(message)
